Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -358,12 +358,6 @@ protected virtual void UpdateCursorTransform()
                 return;
             }
 
-            if (!MixedRealityToolkit.IsInitialized)
-            {
-                // Wait until the toolkit is initialized before taking action.
-                return;
-            }
-
```

Comment:
Is there an equivalent thing here to do around waiting for the input system to be fully initalized, or can we assume that by this point it should already have been?

Reply:
it already should have been.